<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="widget_left" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QAxWidget" name="mapWidget">
         <property name="control" stdset="0">
          <string>{8856f961-340a-11d0-a96b-00c04fd705a2}</string>
         </property>
         <property name="minimumSize" stdset="0">
          <size>
           <width>800</width>
           <height>0</height>
          </size>
         </property>
         <property name="focusPolicy" stdset="0">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="layoutDirection" stdset="0">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="RegisterAsBrowser" stdset="0">
          <bool>false</bool>
         </property>
         <property name="ToolBar" stdset="0">
          <number>0</number>
         </property>
         <property name="enabled" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_right" native="true">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QAxWidget" name="timewidget">
            <property name="control" stdset="0">
             <string>{8856f961-340a-11d0-a96b-00c04fd705a2}</string>
            </property>
            <property name="maximumSize" stdset="0">
             <size>
              <width>200</width>
              <height>70</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCalendarWidget" name="calendarWidget"/>
          </item>
          <item>
           <widget class="QLabel" name="status">
            <property name="text">
             <string>当前状态：无</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="but_clean">
               <property name="text">
                <string>清除模拟路径</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="but_start">
               <property name="text">
                <string>随机路径模拟</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableView" name="tableView">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>150</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1012</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menusystem">
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>开始</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="actionlogin"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menuuser_manage">
    <property name="title">
     <string>系统管理</string>
    </property>
    <addaction name="actionManageDriver"/>
    <addaction name="actionManageCar"/>
   </widget>
   <widget class="QMenu" name="menumap_switch">
    <property name="title">
     <string>地图切换</string>
    </property>
    <addaction name="actiontraffic"/>
    <addaction name="actionsatellite"/>
    <addaction name="actionroadnet"/>
    <addaction name="actiontheme"/>
   </widget>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menusystem"/>
   <addaction name="menuuser_manage"/>
   <addaction name="menumap_switch"/>
   <addaction name="menuhelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionlogin"/>
   <addaction name="actionexit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionlogin">
   <property name="icon">
    <iconset resource="icores.qrc">
     <normaloff>:/Menu/IconRes/Television-Alt.ico</normaloff>:/Menu/IconRes/Television-Alt.ico</iconset>
   </property>
   <property name="text">
    <string>登陆</string>
   </property>
   <property name="iconText">
    <string>登陆</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="actionexit">
   <property name="icon">
    <iconset resource="icores.qrc">
     <normaloff>:/Menu/IconRes/Power.ico</normaloff>:/Menu/IconRes/Power.ico</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="iconText">
    <string>退出</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </action>
  <action name="actionManageDriver">
   <property name="text">
    <string>驾驶员管理</string>
   </property>
  </action>
  <action name="actiontraffic">
   <property name="text">
    <string>打开/关闭交通路况</string>
   </property>
  </action>
  <action name="actionsatellite">
   <property name="text">
    <string>打开/关闭卫星图</string>
   </property>
  </action>
  <action name="actionroadnet">
   <property name="text">
    <string>打开/关闭路网</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>关于软件</string>
   </property>
  </action>
  <action name="actiontheme">
   <property name="text">
    <string>更换主题</string>
   </property>
  </action>
  <action name="actionManageCar">
   <property name="text">
    <string>车辆管理</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QAxWidget</class>
   <extends>QWidget</extends>
   <header>qaxwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icores.qrc"/>
 </resources>
 <connections/>
</ui>
